lib/test/t_Gibbs_regression.cxx

```diff
@@ -170,11 +162,11 @@ int main(int, char *[])
 
     std::cout << "sample mean=" << x_mu << std::endl;
```

Comment:
You could remove all prints, since you're using IGNOREOUT.

Reply:
it can still be useful when debugging